lang/french.json

```diff
@@ -236,19 +256,174 @@
+    "sydneyhud_hudlist_options_buff_options_composite" : "Buffs/effets composites",
+    "sydneyhud_hudlist_options_buff_options_boosts" : "Boosts",
+    "sydneyhud_hudlist_options_buff_options_boosts_desc" : "Effets de boost de Gage/IA",
+    "sydneyhud_hudlist_not_ignore_buff_combat_medic_desc" : "Aussi inclus dans le buff composite de réduction de dégats",
+    "sydneyhud_hudlist_not_ignore_buff_forced_friendship_desc" : "Buff d'équipe",
+    "sydneyhud_hudlist_not_ignore_buff_hostage_taker_desc" : "Aussi inclus dans le buff composite de regénération de vie",
+    "sydneyhud_hudlist_not_ignore_buff_inspire_boost_cooldown" : "Temps de recharge deu boost de la motivation",
+    "sydneyhud_hudlist_not_ignore_buff_inspire_revive_cooldown" : "Temps de recharge de la réanimation de la motivation",
+    "sydneyhud_hudlist_not_ignore_buff_painkillers_desc" : "Aussi inclus dans le buff composite de réduction de dégats",
+    "sydneyhud_hudlist_not_ignore_buff_quick_fix_desc" : "Aussi inclus dans le buff composite de réduction de dégats",
+    "sydneyhud_hudlist_not_ignore_buff_uppers_cooldown" : "Temps de recharge des remontants",
+    "sydneyhud_hudlist_not_ignore_buff_uppers_cooldown_desc" : "Les debuffs composite, s'afficheront en tant que qu'une partie de l'icone principale de buff même désactivé",
+    "sydneyhud_hudlist_not_ignore_buff_bullseye_cooldown" : "Temps de recharge de Dans l'mille",
+    "sydneyhud_hudlist_not_ignore_buff_die_hard_desc" : "Aussi inclus dans le buff composite de réduction de dégats",
+    "sydneyhud_hudlist_not_ignore_buff_overkill_desc" : "Aussi inclus dans le buff composite de boost de dégats",
+    "sydneyhud_hudlist_not_ignore_buff_shock_and_awe_desc" : "Buff d'équipe",
+    "sydneyhud_hudlist_not_ignore_buff_underdog_desc" : "Aussi inclus dans le buff composite de réduction de dégats et de boost de dégats des armes",
+    "sydneyhud_hudlist_not_ignore_buff_unseen_strike_cooldown" : "Unseen Strike Cooldown",
+    "sydneyhud_hudlist_not_ignore_buff_unseen_strike_cooldown_desc" : "Debuff composite, s'affichera en tant que qu'une partie de l'icone principale de buff même désactivé",
+    "sydneyhud_hudlist_not_ignore_buff_berserker_desc" : "Aussi inclus dans le buff composite de boost de dégats d'armes et de mélée",
     "sydneyhud_hudlist_not_ignore_buff_bloodthirst_basic" : "Soif De Sang (Basique)",
     "sydneyhud_hudlist_not_ignore_buff_bloodthirst_aced" : "Soif De Sang (As)",
     "sydneyhud_hudlist_not_ignore_buff_running_from_death_basic" : "Course Contre La Mort (Basique)",
+    "sydneyhud_hudlist_not_ignore_buff_bloodthirst_basic_desc" : "Aussi inclus dans le buff composite de dégats de mélée",
     "sydneyhud_hudlist_not_ignore_buff_running_from_death_aced" : "Course Contre La Mort (As)",
-    
-    "sydneyhud_hud_tweaks": "Amélioration de l'HUD",
+    "sydneyhud_hudlist_not_ignore_buff_up_you_go_desc" : "Also included in damage reduction composite buff",
```

Comment:
This string is also not translated.

Reply:
I've been a bit bored with copy pasting this common sentence that I've may I've skip one like you can see 😒